### ISSUE TITLE
Impossible to start Memote; raises: "ValueError: Expected object or value"

### ISSUE THREAD

**Issue by bdelepine** [opencobra/memote#364]
#### Problem description

Hello guys,

I tried to install memote in a brand new conda env, with pip on a CentOS 7; the installation went well, but I cannot start memote (CLI or import) without getting a traceback with "ValueError: Expected object or value". What am I missing?

Thank you for your help!

#### Code Sample

I installed memote through: 
```bash
$ conda create -n memote_test python=3
$ source activate memote_test
$ pip install memote
```

Any of those commands will leave the same traceback:
```bash
$ memote -h
$ memote report snapshot iJO1366.xml
$ memote --model iJO1366.xml report --one-time
```

Importing memote as a package (``import memote``) also leaves the same traceback.

The traceback:
```python
Traceback (most recent call last):
  File "/home/XXX/.miniconda3/bin/memote", line 7, in <module>
    from memote.suite.cli.runner import cli
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/memote/__init__.py", line 28, in <module>
    from memote.suite.api import *
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/memote/suite/api.py", line 31, in <module>
    from memote.suite.collect import ResultCollectionPlugin
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/memote/suite/collect.py", line 28, in <module>
    from memote.support.helpers import find_biomass_reaction
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/memote/support/helpers.py", line 47, in <module>
    directory + "/data/met_id_shortlist.json"
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/pandas/io/json/json.py", line 366, in read_json
    return json_reader.read()
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/pandas/io/json/json.py", line 467, in read
    obj = self._get_object_parser(self.data)
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/pandas/io/json/json.py", line 484, in _get_object_parser
    obj = FrameParser(json, **kwargs).parse()
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/pandas/io/json/json.py", line 576, in parse
    self._parse_no_numpy()
  File "/home/XXX/.miniconda3/lib/python3.6/site-packages/pandas/io/json/json.py", line 793, in _parse_no_numpy
    loads(json, precise_float=self.precise_float), dtype=None)
ValueError: Expected object or value
```

### Context

[Packages installed in the conda environment.](https://gist.github.com/bdelepine/0007d4aa219afdb7ffa8298d52216b3f)

**Comment by Midnighter** [comment]
Hi @bdelepine, it seems we failed to distribute the JSON file with the memote package and the testing uses a local source package with the file, too. I will make a fix immediately.

**Comment by Midnighter** [comment]
I just released 0.6.1 and consider this issue solved with that. Can you please confirm @bdelepine and re-open this issue if there is any problem?

**Comment by bdelepine** [comment]
Hi @Midnighter, I confirm that the problem is solved: memote no longer raises the ValueError at start-up. Thank you for such a quick help!
